### ISSUE TITLE
Assertion error when playing streams

### ISSUE THREAD

**Issue by orontee** [suaviloquence/mopidy-listenbrainz#6]
# Context

When playing a stream, I see this assertion error in the logs

```
juin 03 22:38:41 argos mopidy[768]: Traceback (most recent call last):
juin 03 22:38:41 argos mopidy[768]:   File "/usr/local/lib/python3.11/dist-packages/mopidy/listener.py", line 44, in on_event
juin 03 22:38:41 argos mopidy[768]:     getattr(self, event)(**kwargs)
juin 03 22:38:41 argos mopidy[768]:   File "/usr/local/lib/python3.11/dist-packages/mopidy_listenbrainz/frontend.py", line 168, in track_playback_started
juin 03 22:38:41 argos mopidy[768]:     self.lb.submit_listen(
juin 03 22:38:41 argos mopidy[768]:   File "/usr/local/lib/python3.11/dist-packages/mopidy_listenbrainz/listenbrainz.py", line 144, in submit_listen
juin 03 22:38:41 argos mopidy[768]:     assert response.ok
juin 03 22:38:41 argos mopidy[768]: AssertionError
```

# Analysis

Should the track URI scheme be checked to skip streams? An alternative is to skip the request to submit when `artist_name` and `release_name` are empty strings (the case of all the streams I've checked). Third option, simply log a debug message when the request fails with 400 code (and log the response message explaining the reason of the failure).

 @suaviloquence Any recommendation?

**Comment by suaviloquence** [comment]
> An alternative is to skip the request to submit when `artist_name` and `release_name` are empty strings (the case of all the streams I've checked).

I think this is the best option.  We might want to skip when either `artist_name` *or* `release_name` are empty strings, to avoid probably extraneous scrobbles, although we could warn/debug log when this happens.

**Comment by suaviloquence** [comment]
Fixed by #10
